dev/com.ibm.websphere.appserver.org.eclipse.persistence-2.7/com.ibm.websphere.appserver.org.eclipse.persistence-2.7.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.persistence-2.7
+singleton=true
+IBM-Process-Types: server, \
+ client
+-bundles=com.ibm.websphere.appserver.thirdparty.eclipselink.2.7; apiJar=false; location:=dev/api/third-party/, \
```

Comment:
we also want to provision `-feature=com.ibm.websphere.appserver.javax.persistence.base-2.2` here